panel/src/components/Forms/Writer/Writer.vue

```diff
@@ -127,6 +126,11 @@ export default {
       toolbar: false
     };
   },
+  computed: {
+    isParagraphNodeHidden() {
+      return Array.isArray(this.nodes) === true && this.nodes.length !== 3 && this.nodes.includes("paragraph") === false;
```

Comment:
This means: do not load paragraph node if the loaded nodes are not default and do not contain paragraphs.